meta-ostro/conf/distro/include/ostro-os-production.inc

```diff
@@ -2,5 +2,13 @@
 # in local.conf. In contrast to ostro-os-development.inc, no fallback
 # is provided.
 
+# Prefer systemd way of creating getty symlinks (instead of
+# the default systemd-serialgetty that creates everything in
+# do_install).
+PACKAGECONFIG_append_pn-systemd = "serial-getty-generator"
```

Comment:
Ah, you are using the same systemd change as my patch.

But I think it is better to have it applied distro-wide, to minimize changes between images.